content/html/concepts/tags/terms/dt/dt.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '<dt>'
+Description: ''
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Lists'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<dt>` tag describes a term found inside a `<dl>` tag. It usually comes with at least one corresponding `<dd>` details tag.
+
+## Syntax
+
+```html
+<dl>
+  <dt>Term</dt>
+  <!-- <dd> detail(s) here -->
+</dl>
+```
+
+In order to render correctly, the `<dt>` tag must be placed inside a `<dl>` tag.
+
+## Example
+
+In the example below, the `<dt>` is used (and highlighted with some styling):
+
+```html
```

Comment:
Again, might benefit from using some more real-world terms/details for the example.